torch/_inductor/codegen/cpp.py

```diff
@@ -804,6 +805,23 @@ def atanh(x):
     def copysign(x, y):
         return f"std::copysign({x}, {y})"
 
+    @staticmethod
+    def frexp(x):
+        cache_keys = f"frexp({x})[0]", f"frexp({x})[1]"
+        if all(cache_key in V.kernel.cse.cache for cache_key in cache_keys):
+            return tuple(V.kernel.cse.cache[cache_key] for cache_key in cache_keys)
```

Comment:
Why was this change needed?

Reply:
https://github.com/pytorch/pytorch/blob/d9db9e62e3d2d58d4e76a43f30c15db389e51c17/torch/_inductor/codegen/cpp.py#L1561 assumes that the cache values are variables and not tuples. I can try to change that function to not assume that if that's preferred.

Reply:
Hrm okay, a bit less elegant but oh well